packages/genui/a2ui-playground/src/styles.css

```
@@ -354,6 +396,68 @@ a {
+  padding: 2px 8px;
+  border-radius: 4px;
+  background: var(--geist-surface);
+  border: 1px solid var(--geist-border);
+  font-size: 11px;
+  font-weight: 500;
+  color: var(--geist-foreground);
+  white-space: nowrap;
+  animation: tagAppear 300ms ease-out;
+}
+
+@keyframes tagAppear {
```

Stylelint (17.9.0)

[error] Expected keyframe name "tagAppear" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)